minifier.js

```diff
@@ -13,12 +13,20 @@ function minifyHTML(opts) {
             if (typeof callback === 'undefined') {
                 return function (err, html) {
                     if (err) {
-                        console.error(err)
-                        if (opts.displayErrors === true) {
-                            res.send('Rendering error: ' + err.message);
-                        } else {
-                            res.sendStatus(500);
+                        if (typeof opts.errCallback === 'function') //custom error callback specified
+                        {
```

Comment:
Could you put this curly bracket on the same line as the ending parenthesis and put the comment above that?